### PR TITLE
Release Notes 2.0.0

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
+# OP-TEE - version 2.0.0
+
+
+## New features
+
+* Generic driver: A new generic TEE driver is in the process of being
+  [upstreamed][gendrv_v8].
```
Comment (jenswi-linaro) [inline CHANGELOG.md]: v9 is out now, https://lkml.org/lkml/2016/4/1/205

**Comment by pascal-brand-st-dev** [comment]
updated

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+| Standard tests | The [optee_test][optee_test] project. |
+| Extended tests | optee_test with tests from the GlobalPlatform™ TEE Initial Configuration Test Suite v1.1.0.4. |
+
+*	ARM Juno Board (vexpress-juno), standard.
+*	Foundation Models (vexpress-fvp), standard tests + extended tests,
+	using FVP ARM V8 Foundation Platformr0p0 (platform build 9.5.40)
+*	HiKey (hikey), standard.
+*	QEMU (vexpress-qemu), standard + extended tests.
+*	STM Cannes (stm-cannes), standard + extended tests.
+
```
Comment (jbech-linaro) [inline CHANGELOG.md]: Please also add:
```MTK8173-EVB (mediatek-mt8173), standard.```

**Comment by jbech-linaro** [comment]
Just one minor comment about missing MTK device.
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Pull request merged**